### DISCUSSION TITLE
Want to open Webview view on click of notification action.

### DISCUSSION THREAD

**Discussion by amitjoshi438** [microsoft/vscode-discussions#689] [Extension Development QnA] [answered]
I have a ` vscode.WebviewViewProvider` panel registered in activity bar and want to show an actionable notification to user on first load of extension which will open the webview view panel. Below is the snippet for the same, but as per my understanding the `webview.show()` will only work once the webview panel is opened once. Which defies the purpose of notification, what is the right way to achieve this:

@joaomoreno @TylerLeonhardt 

```
function showNotificationForChat(chatProvider:any) {
    const message = vscode.l10n.t('Get help for your work');
    const actionTitle = vscode.l10n.t('Try Chat');

    vscode.window.showInformationMessage(message, actionTitle).then((selection) => {
        if (selection === actionTitle) {
            chatProvider.show();
        }
    });
}
```

**Comment by joaomoreno** [comment]
cc @mjbvz

**Comment by r3m0t** [comment, marked as answer]
If there's a webview in the extension's `package.json`, it automatically creates commands with ID `viewTitleId.viewId.focus` and title `Focus on _ View`, you can use `executeCommand` with the command ID.

**Reply by burkeholland** [reply]
@amitjoshi438 does this fix your issue?
